### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by antoniojgage**
Hey this is pull request to change the readme PLease review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # jakesEateryFlex
 ![image](https://user-images.githubusercontent.com/7440005/79818138-a9ef3200-834c-11ea-9ab1-3febfe99df21.png)
+Hey class we're going to do this activity!
```
Comment (kholder777) [inline README.md:3]: looks fine to me.

**Comment by steveo9219** [comment]
comment

**Comment by pagevanhoy** [comment]
checking the comments

**Comment by hoontank** [comment]
code is good

**Comment by tan-x** [comment]
i don't know man...

**Comment by andrewvenson** [comment]
Test Comment

**Comment by Cranekick50** [comment]
what is going on here?

**Comment by marlon20m** [comment]
Hi im here !

**Comment by adrian-palacios** [comment]
Hi

File: README.md (hunk 1 above)
Comment (juanlajara) [inline README.md:3]: What~!

**Review by steppinonland — APPROVED** [review]
hi i'm leaving a comment antonio!!!!!

**Comment by CGMcBride** [comment]
confusing code

**Comment by walterjamesperry** [comment]
42 42 42 42 42 42 42

**Comment by KEDuran** [comment]
test

**Comment by marlon20m** [comment]
How are u guys so fast?

**Comment by jordanalexis6** [comment]
comment

**Comment by vlt44** [comment]
test

**Comment by caz93** [comment]
comment

**Comment by dianecandler** [comment]
This is interesting and probably something I get to watch tomorrow

**Comment by julietg19** [comment]
:)

**Comment by Steve-Munoz** [comment]
hey!

**Comment by steppinonland** [comment]
normal tuesday night!

**Comment by tan-x** [comment]
thanks for all the fish

**Comment by chong0810** [comment]
Hello

File: README.md (hunk 1 above)
Comment (JDMartinez1531) [reply README.md:3]: maybe sleep on it...

**Comment by seanvhussey** [comment]
comment comment comment

**Review by tan-x — APPROVED** [review]
go for it

**Review by julietg19 — APPROVED** [review]

**Review by juanlajara — COMMENTED** [review]
Whatt~~!

File: README.md (hunk 1 above)
Comment (hoontank) [inline README.md:3]: spelled activity right

Comment (KEDuran) [inline README.md:3]: good to go

Comment (JDMartinez1531) [inline README.md:3]: great speeling

Comment (pagevanhoy) [inline README.md:3]: checking on this activity

Comment (dianecandler) [inline README.md:3]: activity is spelled this way activity

Comment (steveo9219) [inline README.md:3]: **WOW!**

Comment (adrian-palacios) [inline README.md:3]: Let's do it!

Comment (Steve-Munoz) [inline README.md:3]: can't wait!

Comment (CGMcBride) [inline README.md:3]: this is fun

**Review by JDMartinez1531 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (marlon20m) [inline README.md:3]: it is we are*

Comment (andrewvenson) [inline README.md:3]: cavities

Comment (kholder777) [reply README.md:3]: i believe it's activitivities*

Comment (julietg19) [inline README.md:3]: hmmmm

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # jakesEateryFlex
 ![image](https://user-images.githubusercontent.com/7440005/79818138-a9ef3200-834c-11ea-9ab1-3febfe99df21.png)
```
Comment (steppinonland) [inline README.md:2]: AWESOME NAME!! WOOOT! ferris bueller you're my hero!!

File: README.md (hunk 1 above)
Comment (marlon20m) [inline README.md:3]: I give you 100 perfect sentence.

Comment (vlt44) [inline README.md:3]: test

Comment (steppinonland) [inline README.md:3]: ferris bueller you're my hero!!!!!

**Review by steppinonland — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ystamaritq) [reply README.md:3]: test

Comment (antoniojgage) [reply README.md:3]: file_from_ios-2

Comment (antoniojgage) [reply README.md:3]: file_from_ios
giphy copy

Comment (antoniojgage) [reply README.md:3]: froFry

**Review by joshuabogart — COMMENTED** [review]
finally found it

**Review by kholder777 — APPROVED** [review]

**Review by dianecandler — APPROVED** [review]

**Review by CGMcBride — COMMENTED** [review]
approved

**Pull request merged**